upflux-client/src/features/PasswordSettings/PasswordSettings.tsx

```diff
@@ -0,0 +1,129 @@
+import React, { useState } from 'react';
+import { TextInput, Button, Text, Stack, Box, Group } from '@mantine/core';
+import './passwordSettings.css';
+
+export const PasswordSettingsContent: React.FC = () => {
+  // State for form values and errors
+  const [currentPassword, setCurrentPassword] = useState('');
+  const [newPassword, setNewPassword] = useState('');
+  const [confirmPassword, setConfirmPassword] = useState('');
+  
+  const [errors, setErrors] = useState({
+    newPassword: '',
+    confirmPassword: '',
+  });
+
+  // Validation function for the New Password
```

Comment:
Create a file in PasswordSettings Folder 

PasswordSettingsConsts.ts

in that 

```
export const MIN_PASSWORD_LENGTH = 12;
export const HAS_NUMBER_REGEX = /\d/;
... etc
export const PASSWORD_SET_NOT_ENOUGH_LENGTH = "Password must be at least 12 ....";
etc
```

do same for all "magic numbers" and "strings".